src/js/pages/Login.js

```diff
@@ -0,0 +1,23 @@
+import React, { PropTypes as T } from 'react'
```

Comment:
Can we maybe name proptypes as something more descriptive then T? Maybe expand it to Types, or just leave it as proptypes. If its too long to type we can expand it with `const {object, number, etc} = PropTypes`